javascript-custom-rules/src/main/java/org/sonar/samples/javascript/checks/ForbiddenFunctionUseCheck.java

```diff
@@ -39,17 +36,19 @@
  * <p>
  * Those methods can be overridden to process information
  * related to node and issues can be created via {@link DoubleDispatchVisitorCheck#addIssue} methods}.
+ *
+ * <p>
+ * Description can either be given via {@link Rule} annotation or through HTML name <ruleKey>.html loaded in
+ * {@link org.sonar.samples.javascript.JavaScriptCustomRulesDefinition}
+ * </p>
  */
 @Rule(
   key = "S1",
   priority = Priority.MAJOR,
   name = "Forbidden function should not be used.",
   tags = {"convention"}
-// Description can either be given in this annotation or through HTML name <ruleKey>.html located in package src/main/resources/org/sonar/l10n/javascript/rules/<repositoryKey>
 // description = "<p>The following functions should not be used:</p> <ul><li>foo</li> <li>bar</li></ul>",
-  )
-@SqaleSubCharacteristic(RulesDefinition.SubCharacteristics.DATA_RELIABILITY)
-@SqaleConstantRemediation("5min")
```

Comment:
should we mention something about sqale? it should be provided manually now, right?

Reply:
afaik sqale was removed completely and now we have only remediation functions. I think we should not overcomplicate thinks. People will ask once they will need the feature